CMakeLists.txt

```diff
@@ -7,6 +7,19 @@ set(CMAKE_CXX_EXTENSIONS OFF)
 
 set(LINK_LIBS "")
 
+# Use the package PkgConfig to detect GTK+ headers/library files
```

Comment:
Make sure to rebase on the latest master, I believe Erick moved this into the Linux-specific block

Reply:
Erick's PR is not merged yet (https://github.com/scrum-gang/neutralinojs/pull/17) as it depends on mines!